exporter/redisPwd.go

```diff
@@ -0,0 +1,25 @@
+package exporter
```

Comment:
Thanks for putting this in a separate file!
Can you rename it to `pwd_file.go` please?